Sources/BraintreePaymentFlow/BTLocalPaymentRequest.swift

```diff
@@ -223,27 +230,31 @@ extension BTLocalPaymentRequest: BTPaymentFlowRequestDelegate {
                 "sessionId": metadata?.sessionID
             ]
             
-            paymentFlowClientDelegate?.apiClient().post("/v1/payment_methods/paypal_accounts", parameters: params, completion: { [weak self] body, response, error in
```

Comment:
I didn't understand why there was need to do this when apiClient reference is passed into this function. To show separation of concern?